components/expansion-panel/src/vwc-expansion-panel-base.ts

```diff
@@ -1,8 +1,19 @@
 import { LitElement, property } from 'lit-element';
 import { observer } from '@material/mwc-base/observer.js';
+import type { Layout } from '@vonage/vvd-foundation/constants.js';
+
+type ExpantionPanelLayout = Extract<
+	Layout,
+	Layout.Outlined
```

Comment:
the default is ghost right?
```suggestion
	Layout.Outlined,
	Layout.Ghost
```